core/new-gui/src/app/workspace/component/result-panel/console-frame/console-frame.component.ts

```diff
@@ -106,5 +122,13 @@ export class ConsoleFrameComponent implements OnInit, OnChanges {
 
   displayConsoleMessages(operatorId: string) {
     this.consoleMessages = operatorId ? this.workflowConsoleService.getConsoleMessages(operatorId) || [] : [];
+    console.log("got messages", this.consoleMessages);
+  }
+
+  workerIdToAbbr(workerId: string) {
+    const tokens = workerId.split("-");
+    const abbr = "Worker-" + tokens.at(tokens.length - 1);
+    this.workerAbbrToLongNameMapping.set(abbr, workerId);
```

Comment:
do we really need this as class variable? I don't see it being used anywhere

Reply:
the actual workerId from engine looks like `WF3-Projection-operator-80903604-6737-41e5-b473-e86b07b397b8-main-0`, where the last number `0` is the workerIndex for this operator. Here for display purpose, we use the index to show user abbreviated name `Worker-0` on the UI.

Later when user needs to refer to `Worker-0` (e.g., send a debug command to worker 0), we need to find the actual long workerId back so that engine understands the reference. This mapping is to keep reference from `Worker-0` to `WF3-Projection-operator-80903604-6737-41e5-b473-e86b07b397b8-main-0`.

Reply:
actually I can just tune the display to show abbr without changing the workerId value. I will use this method instead and remove the mapping.